cmake/Phylanx_HighFive.cmake

```diff
@@ -7,7 +7,7 @@
 macro(phylanx_setup_hdf5)
     if(PHYLANX_WITH_HIGHFIVE)
 
-        find_package(HDF5 NO_MODULE NO_CMAKE_PACKAGE_REGISTRY)
+        find_package(HDF5)
```

Comment:
Do we really have to remove the `NO_CMAKE_PACKAGE_REGISTRY` option?

Reply:
Yes, `NO_CMAKE_PACKAGE_REGISTRY` the `find_package()` fails on my machine

Reply:
ok, thanks